app.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const { getApiTopics } = require("./controllers/app.controllers");
+const app = express();
+
+app.use(express.json());
```

Comment:
What is this doing? Do any of the tests for this ticket require this line of code to pass?